### DISCUSSION TITLE
Diode positions

### DISCUSSION THREAD

**Discussion by kgl2001** [erichelgeson/BlueSCSI-v1#32] [Q&A] [answered]
Hi. Great project. Just one little question... why do the diodes need to be mounted to the rear of the PCB. Sorry if this is already documented. I couldn't see it explained anywhere.

**Comment by erichelgeson** [comment, marked as answer]
To avoid contact with the bluepill as it sits directly over them.

**Reply by kgl2001** [reply]
Thank you for the clarification. I thought that might be the case, but I also wondered the spacers on the header pins might have been big enough to provide the clearance. Obviously not!

**Reply by erichelgeson** [reply]
They may well, though to be safe I put them on the bottom. If you are comfortable with the spacing/clearance you can.
